packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/operations/ClearExecutor.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.view.emf.operations;
+
+import java.util.Collection;
+
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.emf.ecore.EStructuralFeature;
+import org.eclipse.sirius.components.view.emf.operations.api.IClearExecutor;
+import org.springframework.stereotype.Service;
+
+/**
+ * Used to clear a value from an object.
+ *
+ * @author sbegaudeau
+ */
+@Service
+public class ClearExecutor implements IClearExecutor {
+
+    @Override
+    public Object eClear(EObject instance, String featureName) {
+        if (instance.eClass().getEStructuralFeature(featureName) != null) {
```

Comment:
Nitpick: `instance.eClass().getEStructuralFeature(featureName)` could be called only once.

Reply:
I've updated the code